docs/ja/user-guide/configuration/jobconfiguration.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: main
+title: ジョブの設定
+category: User Guide
+menu: menu_ja
+toc:
+- title: ジョブの設定
+  url: "#ジョブの設定"
+- title: イメージ
+  url: "#イメージ"
+- title: ステップ
+  url: "#ステップ"
+- title: 共通設定
+  url: "#共通設定"
+---
+
+# ジョブの設定
+
+ジョブは、全てのビルドでどのようなことを行うのかということの設定です。ジョブの設定は一つの`image`とリストの`steps`、または`template`を含んでいる必要があります。また、`requires`を使用することで、ジョブのトリガーを指定することもできます。パイプラインワークフローを作成するための`requires`の詳しい使用方法は[workflow](/user-guide/configuration/workflow)を参照してください。
+
+#### 例
+
+```
+jobs:
+    main:
+        image: node:6
+        steps:
+            - init: npm install
+            - test: npm test
+    main2: 
+        template: example/mytemplate@stable
+```
+
+## イメージ
+
+`image`の設定はDockerのイメージ、例えば
+ (hub.docker.com)[https://hub.docker.com] からコンテナを指定します。完全なURLを指定することで、カスタムレジストリからイメージを指定することも出来ます。
+
+#### 例
+
+```
+jobs:
+    main:
+        image: my-custom-registry.example.com/myorg/myimage:label
+        steps:
+            - step1: echo hello
+```
+
+## ステップ
+
+ステップはビルド中で実行したいコマンドのリストです。
+次のように設定できます：
+`step_name: step_command`. ステップは設定した順番に実行されます。
+
+#### 例
+
+```
+jobs:
+    main:
+        image: node:8
+        steps:
+            - step_name: step_command --arg1 --arg2 foo
+            - greet: echo Hello
+```
+
+# 共通設定
+
+`shared`の設定は全てのジョブに適用される特殊なジョブの設定です。各ジョブの設定で項目を設定すると、`shared`で設定されている項目を上書きすることができます。
+
+#### 例
+
+下の例では、`image`と`steps`を共通設定で設定して、mainとmain2のジョブで、それぞれ使用されています。
+
+```
+shared:
+    image: node:8
+    steps:
+        - init: npm install
+        - test: npm test
+
+jobs:
+    main:
+        image: node:6
+    main2:
+        steps:
+            - greet: echo hello
+```
+
+上の例は次の例と同じものになります:
+
+```
+jobs:
+    main:
+        image: node:6
+        steps:
+            - init: npm install
+            - test: npm test
+    main2:
+        image: node:8
+        steps:
+            - greet: echo hello
+
+```
+
+### 参考:
+
+- [Annotations](/user-guide/configuration/annotations) - 主にビルドの実行の設定をする際に使用されるフリーフォームのキー/バリューストア
```

Comment:
Note: these links should be fixed to Japanese version after translating these docs.